ocaml-lsp-server/src/custom_requests/req_type_enclosing.ml

```diff
@@ -0,0 +1,224 @@
+open Import
+
+let capability = ("handleTypeEnclosing", `Bool true)
+
+let meth = "ocamllsp/typeEnclosing"
+
+type params =
+  { text_document_position : Lsp.Types.TextDocumentPositionParams.t
+  ; work_done_progress : Lsp.Types.WorkDoneProgressParams.t
```

Comment:
We do nothing with this param. Most standard LSP queries accept it to report progress, but other custom queries does not appear to handle it. If we don't actually make use of it maybe we should simply remove it ?